lib/bamboo/test.ex

```diff
@@ -1,6 +1,8 @@
 defmodule Bamboo.Test do
   @timeout 100
```

Comment:
Changed my mind. Hardcoded at 100ms is fine for now (for assertion). I did add required configuration when refuting and using process_name though.